### PR TITLE
Clean csx samples

### PR DESCRIPTION
**Pull request by MaddyDev**
Simplified the samples structure to follow javascript. 
Cleaned the samples to remove the [Sql()] attribute from the function code.

### CODE REVIEW

**Review by lucyzhang929 — APPROVED** [review]

**Review by VasuBhog — APPROVED** [review]

File: samples/samples-csx/AddProduct/run.csx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+public static Product Run(HttpRequest req, ILogger log, out Product product)
+{
+    log.LogInformation("C# HTTP trigger function processed a request.");
+
+
+    string requestBody = new StreamReader(req.Body).ReadToEnd();
+    product = JsonConvert.DeserializeObject<Product>(requestBody);
+
+    string responseMessage = string.IsNullOrEmpty(product.Name)
```
Comment (Charles-Gagnon) [inline samples/samples-csx/AddProduct/run.csx:20]: Remove this, it's not even used.

File: samples/samples-csx/GetAndAddProducts/run.csx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+using System.Net;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Primitives;
+using Newtonsoft.Json;
+using System.Collections.Generic;
+
+public static Product[] Run(HttpRequest req, ILogger log, IEnumerable<Product> products, out Product[] productsWithIdentity)
+{
+    log.LogInformation("C# HTTP trigger function processed a request.");
```
Comment (Charles-Gagnon) [inline samples/samples-csx/GetAndAddProducts/run.csx:15]: Remove any unnecessary log messages please - it just clutters up the samples.

File: test/Microsoft.Azure.WebJobs.Extensions.Sql.Tests.csproj [hunk 3]
```diff
@@ -67,15 +67,10 @@
...
+      <_CSharpScriptCopyItems Include="Integration\test-csx\**\*.*" />
     </ItemGroup>
-    <Copy SourceFiles="@(_CsxCopyItems)" DestinationFolder="$(OutDir)\SqlExtensionSamples\Csx\%(RecursiveDir)" />
-    <ItemGroup>
-      <_CommonCopyItems Include="..\samples\samples-csx\Common\**\*.*" />
-    </ItemGroup>
-    <Copy SourceFiles="@(_CommonCopyItems)" DestinationFolder="$(OutDir)\SqlExtensionSamples\Common\%(RecursiveDir)" />
-    <Message Text="Copied CSX Samples output to $(OutDir)\SqlExtensionSamples\Csx" Importance="high" />
+    <Copy SourceFiles="@(_CSharpScriptCopyItems)" DestinationFolder="$(OutDir)\SqlExtensionSamples\Csx\%(RecursiveDir)" />
+    <Message Text="Copied CSX Samples output to $(OutDir)\SqlExtensionSamples\CSharpscript" Importance="high" />
```
Comment (Charles-Gagnon) [inline test/Microsoft.Azure.WebJobs.Extensions.Sql.Tests.csproj:74]:
```suggestion
    <Message Text="Copied CSX Samples output to $(OutDir)\SqlExtensionSamples\Csx" Importance="high" />
```

File: test/Microsoft.Azure.WebJobs.Extensions.Sql.Tests.csproj [hunk 4]
```diff
@@ -67,15 +67,10 @@
     <Copy SourceFiles="@(_PythonCopyItems)" DestinationFolder="$(OutDir)\SqlExtensionSamples\Python\%(RecursiveDir)" />
     <Message Text="Copied Python Samples output to $(OutDir)\SqlExtensionSamples\Python" Importance="high" />
     <ItemGroup>
-      <_CsxCopyItems Include="..\samples\samples-csx\InputBindingSamples\**\*.*" />
-      <_CsxCopyItems Include="..\samples\samples-csx\OutputBindingSamples\**\*.*" />
-      <_CsxCopyItems Include="Integration\test-csx\**\*.*" />
+      <_CSharpScriptCopyItems Include="..\samples\samples-csx\**\*.*" />
```
Comment (Charles-Gagnon) [inline test/Microsoft.Azure.WebJobs.Extensions.Sql.Tests.csproj:70]: nit: Rename back to _CsxCopyItems

**Pull request merged**